cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaMirrorMaker2Cluster.java

```diff
@@ -85,7 +85,7 @@ private KafkaMirrorMaker2Cluster(Reconciliation reconciliation, KafkaMirrorMaker
 
         this.connectors = KafkaMirrorMaker2Connectors.fromCrd(reconciliation, resource);
         this.serviceName = KafkaMirrorMaker2Resources.serviceName(cluster);
-        this.loggingAndMetricsConfigMapName = KafkaMirrorMaker2Resources.metricsAndLogConfigMapName(cluster);
+        this.connectConfigMapName = KafkaMirrorMaker2Resources.metricsAndLogConfigMapName(cluster);
```

Comment:
Ditto as per Kafka Connect, what about changing the `metricsAndLogConfigMapName` method in something like just `configMapName`?